staging/publishing/rules.yaml

```diff
@@ -1653,3 +1653,16 @@ rules:
       branch: master
       dir: staging/src/k8s.io/mount-utils
     name: master
+
+- destination: component-helpers
+  library: true
+  branches:
+  - source:
+      branch: master
+      dir: staging/src/k8s.io/component-helpers
+    name: master
+    dependencies:
+    - repository: apimachinery
+      branch: master
+    - repository: api
+      branch: master
```

Comment:
client-go too?

Reply:
In the current form, component-helpers does not depend on client-go. Adding client-go here will make the publishing-bot-verify check fail.

We avoid adding extra deps here to avoid additional lines in the published go.mod file (it renders awkwardly) - https://github.com/kubernetes/kubernetes/pull/79330#discussion_r296615365